feature/gnmi/ate_tests/telemetry_basic_check_test/README.md

```diff
@@ -0,0 +1,155 @@
+        *   /interfaces/interface/state/counters/in-multicast-pkts
+        *   /interfaces/interface/state/counters/in-discards
+        *   /interfaces/interface/state/counters/in-errors
+        *   /interfaces/interface/state/counters/in-fcs-errors
+        *   /interfaces/interface/state/counters/out-unicast-pkts
+        *   /interfaces/interface/state/counters/out-broadcast-pkts
+        *   /interfaces/interface/state/counters/out-multicast-pkts
+        *   /interfaces/interface/state/counters/out-octets
+        *   /interfaces/interface/state/counters/out-discards
+        *   /interfaces/interface/state/counters/out-errors
+
+*   Send the traffic over the DUT.
+
+    *   Check some counters are updated correctly.
+
+*   QoS counters
+
+    *   Send the traffic with all forwarding class NC1, AF4, AF3, AF2, AF1 and
+        BE1 over the DUT
+    *   Check the QoS queue counters exist and are updated correctly
+        *   /qos/interfaces/interface/output/queues/queue/state/transmit-pkts
+        *   TODO:
+            /qos/interfaces/interface/output/queues/queue/state/transmit-octets
+        *   TODO:
+            /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+
+*   Component
+
+    *   Check the following component paths exists
+        *   /components/component/integrated-circuit/state/node-id
+        *   /components/component/state/parent
+
+*   CPU component state
+
+    *   Check the following component paths exists
+        *   (type=CPU) /components/component/state/description
+        *   (type=CPU) /components/component/state/mfg-name
+
+*   Controller card last-reboot-time and reason
+
+    *   Check the following component paths exists
+        *   (type=CONTROLLER_CARD)
+            /components/component[name=<supervisor>]/state/last-reboot-time
+        *   (type=CONTROLLER_CARD)
+            /components/component[name=<supervisor>]/state/last-reboot-reason
+
+*   Software version
+
+    *   Check the following component paths exists for SwitchChip cards.
+        *   /components/component/state/software-version
+
+*   LACP
+
+    *   Check the bundle interface member path and LACP counters and status.
+        *   /lacp/interfaces/interface/members/member
+
+*   AFT
+
+    *   Check the following AFT path exists.
+        *   TODO: /network-instances/network-instance/afts
+
+*   P4RT
+
+    *   Enable p4-runtime.
+    *   configure interface port ID with minimum and maximum uint32 values.
+    *   Check the following path exists with correct interface ID.
+        *   /interfaces/interfaces/state/id
+    *   configure FAP device ID with minimum and maximum uint64 values.
+    *   Check the following path exists with correct node ID.
+        *   /components/component/integrated-circuit/state/node-id
+
+## Config Parameter coverage
+
+No configuration coverage.
+
+## Telemetry Parameter coverage
+
+/interfaces/interface/state/admin-status
```

Comment:
These paths have to be itemized, otherwise they would show up as a jumbled paragraph.

Reply:
Done!